.github/workflows/ci_windows.yaml

```diff
@@ -152,3 +153,15 @@ jobs:
         with:
           name: qBittorrent-CI_Windows-x64_libtorrent-${{ matrix.libt_version }}
           path: upload
+
+      - name: Create installer
+        run: |
+          7z x -o"dist/windows/" "dist/windows/NSISPlugins.zip"
+          makensis /DQBT_DIST_DIR="../../upload/qBittorrent" dist/windows/qbittorrent.nsi
+
+      - name: Upload installer
+        uses: actions/upload-artifact@v3
+        with:
+          name: qBittorrent-CI_Windows-x64_libtorrent-${{ matrix.libt_version }}-setup
```

Comment:
You might just remove the `Upload build artifacts` step since the installer is much smaller in size.
And what about revert the changes in `Prepare build artifacts` step and use this:
```suggestion
      - name: Create installer
        run: |
          7z x -o"dist/windows/" "dist/windows/NSISPlugins.zip"
          makensis /DQBT_DIST_DIR="../../upload" dist/windows/qbittorrent.nsi

      - name: Upload installer
        uses: actions/upload-artifact@v3
        with:
          name: qBittorrent-CI_Windows-x64_libtorrent-${{ matrix.libt_version }}
```
the additional cmake logs shouldn't pose a problem I reckon.

Reply:
To me, they look like they have a different purpose. Many will prefer to avoid using the installer just to test something.

If you care about the size of "regular" artifacts, then that's another issue.

In any case, I would prefer to leave this aspect as it is. This can be discussed later.

Reply:
> the additional cmake logs shouldn't pose a problem I reckon.

My opinion is the opposite. The explicit separation of cmake logs is an improvement in itself even for "regular" artifacts.